lib/kracken/controllers/token_authenticatable.rb

```diff
@@ -14,7 +14,7 @@ def self.included(base)
       end
 
       # NOTE: Monkey-patch until this is merged into the gem
-      def request_http_token_authentication(realm = 'Application')
```

Comment:
Tested it out, your hunch was right: monkey patching with different airty--even with optional parameters--would result in an error. I added a check for the Rails version after spending 20 min trying to get `instance_method().airty()` to work on a module while it is being defined.

 (╥﹏╥)